vignettes/Model.Rmd

```diff
@@ -10,90 +12,365 @@ vignette: >
+
+# Plot results
+plot(seasonality_simulation[,1], rowSums(seasonality_simulation[,All_mos_cols]),
+     ylim = c(0, 200000), type = "l", xlab = "Days", ylab = "Mosquito population size")
+grid()
+### Run simulation using seasonality over previous species distribution
+# Update seasonality parameter set to include species distributions
+params_seasons_species <- set_species(parameters = params_seasons,
+                                      species = list(arab_params, fun_params, gamb_params),
+                                      proportions = c(0.1,0.3,0.6))
+
+# Run simulation
+seasonality_species_simulation <- run_simulation(timesteps = 600,
+                                                 parameters = params_seasons_species)
+
+# Collect results
+Mos_season_species_sim <- cbind(seasonality_species_simulation$timestep,
+                         rowSums(seasonality_species_simulation[,arab_cols]),
+                         rowSums(seasonality_species_simulation[,fun_cols]),
+                         rowSums(seasonality_species_simulation[,gamb_cols]))
+
+# Plot results
+species_plot(Mos_season_species_sim)
+
 ```
+
+The mosquito population size is no longer constant and follows the patterns set by rainfall.
+
+### Individual or deterministic mosquitoes
+
+Mosquitoes may also be modelled deterministically, rather than individually (which is the default).
+
+This can be done by setting `individual_mosquitoes` to `FALSE` in the `overrides` argument of `get_parameters()`.
+
+```{r, fig.align = 'center', fig.height = 5, fig.width = 7}
+simparams <- get_parameters(overrides = list(individual_mosquitoes = FALSE))
+```
+
+## Vignettes
+
+The remaining vignettes describe how to adjust sets of parameters through a number of functions as follows:
+
+1.  [Population Demography](https://mrc-ide.github.io/malariasimulation/articles/Demography.html)
+
+    -   Age group rendering
+    -   `set_demography()`: setting demographies
+    -   `set_equilibrium()`: establishing initial human and mosquito populations to achieve initial equilibrium conditions
+
+2.  [Treatment](https://mrc-ide.github.io/malariasimulation/articles/Treatment.html)
+
+    -   `set_drugs()`: for drug-specific parameters (with in-built parameter sets)
+    -   `set_clinical_treatment()`: implemention of clinical treatment interventions
+
+3.  [MDA and Chemoprevention](https://mrc-ide.github.io/malariasimulation/articles/MDA.html)
+
+    -   `set_mda()`: implementation of mass drug administration interventions
+
+    -   `set_smc()`: implementation of seasonal malarial chemoprevention interventions
+
+    -   `set_pmc()`: implementation of perrenial malarial chemoprevention interventions
+
+    -   `peak_season_offset()`: correlating timed interventions with seasonal malaria
+
+4.  [Vaccines](https://mrc-ide.github.io/malariasimulation/articles/Vaccines.html)
+
+    -   `set_mass_rtss()`: implementation of a RTS,S vaccination intervention
```

Comment:
Can change these two to `set_mass_pev()` and `set_pev_epi()` based on vaccine vignette changes. And change mentions of RTS,S to "malaria vaccine"

Reply:
Changed!